tests/capstone2llvmir/x86_tests.cpp

```diff
@@ -9501,6 +9501,466 @@ TEST_P(Capstone2LlvmIrTranslatorX86Tests, X86_INS_CMOVS_no_move)
 	EXPECT_NO_VALUE_CALLED();
 }
 
+//
+// X86_INS_FCMOVB
+//
+
+TEST_P(Capstone2LlvmIrTranslatorX86Tests, X86_INS_FCMOVB_move_1)
+{
+	ALL_MODES;
+
+	setRegisters({
+						 {X87_REG_TOP, 0x5},
+						 {X86_REG_ST5, 3.14}, // st(0)
+						 {X86_REG_ST6, 15.7}, // st(1)
+						 {X86_REG_CF, true},
+				 });
```

Comment:
Fix indents to match other indents in this file - here, and in all the other code you added - pretty much all `setRegisters()`, `EXPECT_JUST_REGISTERS_STORED()`, etc. are indented too much.
This wile uses tab indentation.